lib/rspec-stripe/test_frameworks/rspec.rb

```diff
@@ -18,13 +18,33 @@ def stripe_customer
               @stripe_customer ||= @runner.customer
             end
 
+            def stripe_customer_event
+              @stripe_customer_event ||= @runner.customer_event
+            end
+
+            def stripe_card_event
+              @stripe_card_event ||= @runner.card_event
+            end
+
             def stripe_plan
               @stripe_plan ||= @runner.plan
             end
 
+            def stripe_plan_event
+              @stripe_plan_event ||= @runner.plan_event
+            end
+
             def stripe_subscription
               @stripe_subscription ||= @runner.subscription
             end
+
+            def stripe_subscription_event
+              @stripe_subscription_event ||= @runner.subscription_event
+            end
+
+            def last_stripe_event
```

Comment:
This feels out of place in this gem. It seems like something that should be called in the consumer spec